src/SparkFun_u-blox_SARA-R5_Arduino_Library.cpp

```diff
@@ -174,15 +174,15 @@ bool SARA_R5::begin(HardwareSerial &hardSerial, unsigned long baud)
 
 //Calling this function with nothing sets the debug port to Serial
 //You can also call it with other streams like Serial1, SerialUSB, etc.
-void SARA_R5::enableDebugging(Stream &debugPort)
+void SARA_R5::enableDebugging(Print &debugPort)
```

Comment:
Hi Michael (@mazgch ),

I will need to test this change. I think it will cause problems on some platforms. It may work on ESP32, but I think will fail on others. I will let you know what I find.

Thank you,
Paul

Reply:
Looks like this is OK. No compile errors to be seen...